src/components/KEmptyState/KEmptyState.vue

```diff
@@ -86,24 +97,35 @@ defineProps({
   padding: 48px 0;
```

Comment:
```suggestion
  padding: var(--spacing-xxl, spacing(xxl)) 0;
```
While you're in this component

Reply:
Ah, nice. I didn’t see that one. There are a couple more things in here that could use custom properties (e.g. the border radius), but I’d rather keep the scope to spacing for the time being.

Changed.